addons/common/fnc_removePerFrameHandler.sqf

```diff
@@ -29,13 +29,19 @@ if (_handle < 0 || {_handle >= count GVAR(PFHhandles)}) exitWith {};
 [{
     params ["_handle"];
 
-    GVAR(perFrameHandlerArray) deleteAt (GVAR(PFHhandles) select _handle);
-    GVAR(PFHhandles) set [_handle, nil];
+    GVAR(perFrameHandlerArray) set [GVAR(PFHhandles) select _handle select 0, {}];
```

Comment:
Fun fact. `call` doesn't exit early if it has no code. It will actually really build a callstack and everything required. And then execute the empty code and exit right after.

Reply:
I wouldn't have implemented it any other way tbh. For simplicity.